src/views/AgmLocationChg.vue

```diff
@@ -14,30 +14,8 @@
       @exit="onPaymentErrorDialogExit()"
     />
 
-    <!-- Initial Page Load Transition -->
-    <v-fade-transition>
-      <div
-        v-show="showLoadingContainer"
-        class="loading-container"
-      >
-        <div class="loading__content">
-          <v-progress-circular
-            color="primary"
-            :size="50"
-            indeterminate
-          />
-          <div class="loading-msg">
-            {{ loadingMessage }}
-          </div>
-        </div>
-      </div>
-    </v-fade-transition>
-
     <!-- Main Body -->
-    <v-container
-      v-if="dataLoaded"
-      class="view-container"
-    >
```

Comment:
@JazzarKarim Since we don't fetch drafts (or load any extra data), we don't need the loading spinner or a conditional on the main body.

Reply:
You're absolutely right. Good catch Sev. I probably should've made this change in one of my PRs.